pkg/serverless/aws/arn.go

```diff
@@ -8,11 +8,12 @@ package aws
 import (
 	"encoding/json"
 	"io/ioutil"
+	"path/filepath"
 	"strings"
 	"sync"
 )
 
-const persistedStateFilePath = "/tmp/dd-lambda-extension-cache.json"
+var persistedStateFilePath = filepath.FromSlash("/tmp/dd-lambda-extension-cache.json")
```

Comment:
This will return `\tmp\dd-lambda-extension-cache.json` which doesn't exist on Windows. It's not a issue of slash vs back-slash but an issue of `tmp` not being a folder on windows.

You need to either use `ioutil.TempDir`/`ioutil.TempFile` or similar, which will return something like to `C:\Users\USERNAME\AppData\Local\Temp\4dd-lambda-extension-cache_63607291` on windows.

Or, If this feature is not suppose to ever run on windows, you can use build tags to prevent it from running on Windows (aka `// +build !windows`). We do this for multiple code path (ex:  [this](https://github.com/DataDog/datadog-agent/blob/master/pkg/api/security/platform_nowindows.go) vs [this](https://github.com/DataDog/datadog-agent/blob/master/pkg/api/security/platform_windows.go))